gef-ui/templates/partials/confirm-cover-start-date.njk

```diff
@@ -35,13 +37,56 @@
         </div>
     </div>
 
-    <h1 data-cy="confirm-cover-start-date-heading">Confirm cover start date for issued facilities</h1>
+    {% if facility.length > 0 %}
+        {% if success %}
+            {{
+                mojBanner({
+                type: 'success',
+                text: success.message,
+                iconFallbackText: 'Success'
+                })
+            }}
+        {% endif %}
+    {% endif %}
 
-    {% if isUkefReviewAvailable %}
+    {% if isUkefReviewPositive %}
         <div class="govuk-grid-row govuk-body">
             <div class="govuk-grid-column-two-thirds">
-                <form method="POST" autocomplete="off">
-                </form>
+                {% if facility.length > 0 %}
+                    {{ govukTable
+                        ({
+                                caption: "Confirm cover start date for issued facilities",
```

Comment:
minor thing, could we get the indentation to be consistent with everything else? :)

Reply:
Done